aws/resource_aws_ses_configuration_set.go

```diff
@@ -78,29 +79,27 @@ func resourceAwsSesConfigurationSetDelete(d *schema.ResourceData, meta interface
 	return err
 }
 
-func findConfigurationSet(name string, token *string, meta interface{}) (bool, error) {
+func findConfigurationSet(name string, meta interface{}) (bool, error) {
 	conn := meta.(*AWSClient).sesConn
 
 	configurationSetExists := false
 
-	listOpts := &ses.ListConfigurationSetsInput{
-		NextToken: token,
+	configSetInput := &ses.DescribeConfigurationSetInput{
+		ConfigurationSetName: aws.String(name),
 	}
 
-	response, err := conn.ListConfigurationSets(listOpts)
-	for _, element := range response.ConfigurationSets {
-		if *element.Name == name {
-			configurationSetExists = true
-		}
-	}
-
-	if err != nil && !configurationSetExists && response.NextToken != nil {
-		configurationSetExists, err = findConfigurationSet(name, response.NextToken, meta)
-	}
+	response, err := conn.DescribeConfigurationSet(configSetInput)
 
 	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok && aerr.Code() == "ConfigurationSetDoesNotExist" {
```

Comment:
hi @carthewd! while this is logically correct, in the provider we're transitioning to using the `isAwsError` method so to consolidate this line we can do something like
```suggestion
    if isAWSErr(err, ses.ErrCodeConfigurationSetDoesNotExistException, "") {}
```